superset/connectors/sqla/models.py

```diff
@@ -1326,27 +1360,33 @@ def query(self, query_obj: QueryObjectDict) -> QueryResult:
         errors = None
         error_message = None
 
-        def mutator(df: pd.DataFrame) -> None:
+        def assign_column_label(df: pd.DataFrame) -> None:
```

Comment:
looks like this return type is wrong, it should at least be an `Optional[pd.DataFrame]`

Reply:
Good catch!